src/CarrierLogo/CarrierLogo.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View, Image, PixelRatio } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import StyleSheet from '../PlatformStyleSheet';
+import type { Props } from './CarrierLogoTypes';
+import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS, BASE_URL } from './consts';
+import { logoSizes } from './styles';
+
+export default function CarrierLogo({
+  size = SIZE_OPTIONS.LARGE,
+  carriers = [],
+}: Props) {
+  const carriersLength = carriers.length;
+  const urlSize =
+    (carriersLength > 1 || size === SIZE_OPTIONS.SMALL ? 16 : 32) *
+    (PixelRatio.get() >= 2 ? 2 : 1);
+
+  return (
+    <View
+      style={[
+        styles.wrapper,
+        carriersLength > 1 ? sizeStyles[SIZE_OPTIONS.LARGE] : sizeStyles[size],
+      ]}
+    >
+      {carriers.slice(0, 4).map((carrierImage, index) => (
+        <Image
+          key={carrierImage.code}
+          style={[
+            styles.logo,
+            getLogoSizeStyle(carriersLength, size),
+            carriersLength === 2 && index === 1 && styles.logoFlexEnd,
+          ]}
+          source={{
+            uri: `${BASE_URL}/airlines/${urlSize}/${
```

Comment:
Could you extract this URL generation into a function and write a test for it too?

Reply:
Also, maybe in a future iteration, we would want these assets to be available as assets for mobile apps, when on the web they could be fetched from URL. WDYT?

Reply:
Sure, I'll extract it. 
And we can definitely extend it later and bundle the assets. I'm just not sure how often assets like carrier logos changes and expands.

Reply:
Helper extracted and tests added.